src/components/InventoryGroups/Modals/RemoveHostsFromGroupModal.js

```diff
@@ -55,8 +55,8 @@ const RemoveHostsFromGroupModal = ({
 
     apiWithToast(
       dispatch,
-      () =>
-        removeHostsFromGroup(
+      async () =>
+        await removeHostsFromGroup(
```

Comment:
I still can see the deleted host after I submit the deletion modal:
1. Select any host,
2. Delete it and wait for the table to be reloaded
3. The deleted host is still present in the table. However, if I do refresh again, it will disappear. It is most probably caused by the backend delay.

Reply:
There are more ways of how to tackle this: we can delay the inventory table reload so that in most of the cases, users will see the accurate data in the table. Or, we can fake the data: don't request hosts again, but just remove the deleted host from the list "manually in UI".

Reply:
For the first option, this works well for me (but yeah, it creates some delay in response to users):

`delay(() => inventory.current.onRefreshData({}, false, true), 1000);` where delay is imported from lodash. But `setTimeout` can be used here too as well.

Reply:
I think this is caused by React 18 and their batch api calls. Usually using a useEffect that looks for something that the first api call changes, and then makes the second api call is the way to go. I'm just having trouble reloading the table after the `removeHostsFromGroupModalOpen` is set to false.

Reply:
I think this is caused by React 18 and their batch api calls. Usually using a useEffect that looks for something that the first api call changes, and then makes the second api call is the way to go. I'm just having trouble reloading the table after the removeHostsFromGroupModalOpen is set to false.

Reply:
I have pushed a new commit where I am trying to make it possible to change modal reload timeout.